pkg/virtctl/root.go

```diff
@@ -36,11 +37,21 @@ func NewVirtctlCommand() *cobra.Command {
 	}
 
 	// used in cobra templates to display either `kubectl virt` or `virtctl`
-	cobra.AddTemplateFunc("ProgramName", func() string { return programName })
+	cobra.AddTemplateFunc(
```

Comment:
@slintes I made a few probes on the compiled binary regarding the help templating, all probes still worked as expected.

Reply:
perfect, thanks!